tests/integration/annotation_import/conftest.py

```diff
@@ -119,18 +130,6 @@ def text_data_row(rand_gen):
     }
 
 
-@pytest.fixture()
```

Comment:
moved into main conftest, because I needed to use it on that level... this I should straighten out out when I work on DICOM